doc/api/util.md

```diff
@@ -1425,8 +1425,10 @@ changes:
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
     * `default` {string | boolean | string\[] | boolean\[]} The default option
-      value when it is not set by args. It must be of the same type as the
-      `type` property. When `multiple` is `true`, it must be an array.
+      value when this argument is not passed. It must be of the same type as the
+      `type` property. When `multiple` is `true`, it must be an array. For
+      `string` arguments, if the option is passed it must still be followed by
+      a value.
```

Comment:
```suggestion
      `type` property. When `multiple` is `true`, it must be an array.
```

In my personal opinion, saying `"The default option value when this argument is not passed."` implies that if it passed, it must have a value.

Secondly, `"it must still be followed by a value."` makes it sound like the `"default"` option doesn't work on string arguments...

Reply:
> makes it sound like the `"default"` option doesn't work on string arguments.

The `default` option doesn't do anything when the option is passed (except in `strict: false`), so that's a correct inference (as long as you don't skip over the "if the option is passed" part).

Reply:
I think "it must still be followed by a value" is redudant because of "The default option value when this argument is not passed.", but that might just be me.